go.mod

```diff
@@ -1,41 +1,43 @@
 module kubevirt.io/ssp-operator
 
-go 1.24.3
+go 1.24.4
+
+toolchain go1.24.5
```

Comment:
https://go.dev/doc/toolchain#work mentions

> To remove the toolchain line from a go.work file, use go work edit -toolchain=none.

Does it mean it might be possible to get rid of the toolchain line using golang tools?

Reply:
We will need to add workspaces to SSP to do that. Maybe we should, but I'm not sure if in this case there is any benefit. WDYT @0xFelix?

Reply:
It is most likely better to simply bump the `go` directive.

Reply:
I don't think the main issue will go away by using workspaces.